pom.xml

```diff
@@ -31,13 +31,24 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins.version>1.590</jenkins.version>
         <java.level>6</java.level>
         <workflow.version>1.4.2</workflow.version>
     </properties>
 
     <dependencies>
         <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-support</artifactId>
+            <version>${workflow.version}</version>
```

Comment:
I need this dependency for the `RunWrapper` object. But it is possible to work even if I provide it as an optional dependency.